package.json

```diff
@@ -34,6 +34,7 @@
     "express-session": "^1.17.1",
     "helmet": "^4.2.0",
     "hpp": "^0.2.3",
+    "html-entities": "^1.4.0",
```

Comment:
This will be needed in sanitizing comments section, so the user won't be able to paste i.e. <script> tag into comment, not neccesarily here.